matchms/filtering/derive_adduct_from_name.py

```diff
@@ -2,15 +2,16 @@
 from ..utils import looks_like_adduct
 
 
-def derive_adduct_from_name(spectrum_in: SpectrumType, remove_adduct_from_name=True) -> SpectrumType:
+def derive_adduct_from_name(spectrum_in: SpectrumType,
+                            remove_adduct_from_name: bool = True) -> SpectrumType:
     """Find adduct in compound name and add to metadata (if not present yet).
 
     Method to interpret the given compound name to find the adduct.
     Args:
     ----
```

Comment:
Does not render correct in readthedocs
```suggestion
    Parameters
    ---------------
```